src/hub/views.py

```diff
@@ -235,92 +235,6 @@ def invite_to_hub(self, request, pk=None):
 
         return Response(response, status=200)
 
-    @action(detail=True, methods=[GET])
-    def latest_actions(self, request, pk=None):
```

Comment:
Not sure why this shows as removed in your branch. I think it has to do with some work we recently merged unless you also did related work on this? Not sure if this change is related to your branch